src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -1822,7 +1822,7 @@ protected void AfterMembersChecks(BindingDiagnosticBag diagnostics)
             var location = GetFirstLocation();
             var compilation = DeclaringCompilation;
 
-            if (this.IsRefLikeType || this.IsExtension)
```

Comment:
>e || this.IsExtension [](http://example.com/codeflow?start=33&length=21)

This is probably a good change regardless of the attribute change because extension types are not ref-types.

Reply:
Yes, I'd missed that in previous PR